doc/release-notes.md

```diff
@@ -17,3 +27,9 @@ Regular backups are still necessary, however, in order to ensure that
 transparent and Sprout addresses are not lost.
 
 [Pull request](https://github.com/zcash/zcash/pull/3492), [ZIP 32](https://github.com/zcash/zips/blob/master/zip-0032.mediawiki)
+
+Fix Signing Raw Transactions with Unsynced Offline Nodes
+--------------------------------------------------------
+With v2.0.0, in `signrawtransaction` the consensus branch ID (which is used to construct the transaction) was estimated using the chain height. With v2.0.1 this has been improved by also considering the `APPROX_RELEASE_HEIGHT` of the release, and a new parameter to allow the caller to manually override the consensus branch ID that zcashd will use.
```

Comment:
Is it true that 2.0.0 uses estimated chain height? I don't remember that.

Reply:
Yes, for never synced nodes, this meant 0. https://github.com/zcash/zcash/pull/3520/files#diff-01aa7d1d32f1b9e5a836c9c411978918L877